src/metatrain/soap_bpnn/trainer.py

```diff
@@ -300,6 +303,12 @@ def train(
                     targets, (model.module if is_distributed else model).scaler
                 )
                 systems, targets = systems_and_targets_to_dtype(systems, targets, dtype)
+
+                extra_data = tensormap_to_device(
+                    tensormap_dict=extra_data, device=device
+                )
+                extra_data = tensormap_to_dtype(tensormap_dict=extra_data, dtype=dtype)
```

Comment:
Isn't this function only one line?
```
extra_data = {k: v.to(device) for k, v in extra_data.items()}
```
In that case, shouldn't it be better to be more explicit and avoid this function definition?

Or, even better, should we include it in `systems_and_targets_to_dtype`?